bx_py_utils_tests/tests/test_test_utils_snapshot.py

```diff
@@ -0,0 +1,20 @@
+import pathlib
+import tempfile
+
+import pytest
+
+from bx_py_utils.test_utils.snapshot import assert_snapshot
+
+
+def test_assert_snapshot():
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        with pytest.raises(FileNotFoundError) as excinfo:
+            assert_snapshot(tmp_dir, 'snap', [{'foo': 42, 'bar': 5}])
```

Comment:
add a non ascii example, too.

Reply:
Updated the boring `bar` to an exciting `bär`